core/trie/trie.go

```diff
@@ -57,16 +57,16 @@ func RunOnTempTrie(height uint, do func(*Trie) error) error {
 // FeltToBitSet Converts a key, given in felt, to a bitset which when followed on a [Trie],
 // leads to the corresponding [Node]
 func (t *Trie) FeltToBitSet(k *felt.Felt) *bitset.BitSet {
```

Comment:
We should make this private as well

Reply:
I think the `FeltToBitSet()` is a very useful tool function, otherwise, it's hard to convert a `Felt` to a `Bitset`, as we can't get the Trie's height from out side of Trie package.
And I also make it private in  [commit](https://github.com/NethermindEth/juno/pull/577/commits/1626544d6169f4677d6733166b16e7d74f025372).

Reply:
Yes, however, it is not being used outside of the package anywhere. With regards to its utility, it would be better to make it a static function instead of a method on a trie struct. For our use case, this is not necessary.

Reply:
Ohh, I see